### PR TITLE
chore: notice for https://github.com/aws/aws-cdk/issues/26325

### PR DESCRIPTION
**Pull request by iliapolo**
Fixes #

### CODE REVIEW

File: data/notices.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
   "notices": [
+    {
+      "title": "custom resources: provisioning failures",
+      "issueNumber": 26325,
+      "overview": "Custom resources may fail to deploy / destroy ",
+      "components": [
+        {
+          "name": "aws-cdk-lib.core.CustomResource",
```
Comment (iliapolo) [inline data/notices.json:9]: Not sure we can get any more specific that this...can we?

Comment (mrgrain) [reply data/notices.json:9]: Does this cover all the cases?

Comment (iliapolo) [reply data/notices.json:9]: I think so. 

**eks: cluster-resource-handler onDelete fails for fargate and cluster events**

- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-eks/lib/fargate-profile.ts#L177
- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-eks/lib/cluster-resource.ts#L70

**s3: autoDeleteObjects fails when bucket doesn't exist on resource deletion**

- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-s3/lib/bucket.ts#L2451

**core: Custom::CrossRegionExportWriter fails with InvalidResourceId: UnknownError**

- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/core/lib/custom-resource-provider/cross-region-export-providers/export-writer-provider.ts#L113

**integ-tests-alpha: assertions provider function is not working**

- https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/integ-tests-alpha/lib/assertions/assertions.ts#L52

**custom-resource: ignoreErrorCodesMatching broken on AwsCustomResource with Node.js 18/AWS SDK v3**

- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts#L460

**@aws-cdk/aws-redshift-alpha: Custom resource unable to find AWS SDK v3**

- https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-redshift-alpha/lib/private/database-query.ts#L60

**redshift-alpha: redshift cluster-reboot fails**

- https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-redshift-alpha/lib/cluster.ts#L733

Comment (iliapolo) [reply data/notices.json:9]: The problem is it also covers plenty of non broken cases...but I don't think we can help that.

**Review by mrgrain — APPROVED** [review]

**Pull request merged**